.travis.yml

```diff
@@ -3,80 +3,35 @@ sudo: required
 git:
   quiet: true
 
-env:
-  global:
-    - ROS_REPO=ros
-    - CMAKE_ARGS="-DENABLE_TESTS=ON"
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=dependencies.rosinstall
-
 jobs:
-  include:
-    - os: linux
-      dist: trusty
-      language: cpp
-      env:
-      - ROS_DISTRO="kinetic"
-      - CCACHE_DIR=$HOME/.ccache
-      - BADGE=trusty
-      cache:
-        directories:
-          - $HOME/.ccache
-    - os: linux
-      dist: xenial
-      language: cpp
-      env:
-      - ROS_DISTRO="melodic"
-      - CCACHE_DIR=$HOME/.ccache
-      - BADGE=xenial
-      cache:
-        directories:
-          - $HOME/.ccache
-    - os: linux
-      dist: xenial
-      language: cpp
-      env:
-      - ROS_DISTRO="melodic"
-      - CCACHE_DIR=$HOME/.ccache
-      - BADGE=xenial-clang-compiler
-      - ADDITIONAL_DEBS="clang"
-      - CMAKE_ARGS="-DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DENABLE_TESTS=ON"
-      cache:
-        directories:
-          - $HOME/.ccache          
+  include:       
     - os: windows
```

Comment:
Does Github Actions not support Windows builds? It seems a bit odd to maintain two separate CI builds on different utilities

Reply:
It probably does, but I am not sure I have time to go through the process at the moment.